app/javascript/components/Drawing.jsx

```diff
@@ -0,0 +1,84 @@
+import React, { Component } from 'react';
+import { Image } from 'react-konva';
+
+export default class Drawing extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      isDrawing: false,
+    } 
+  }
+
+  componentDidMount() {
+    const canvas = document.createElement("canvas");
+    canvas.width = 800;
+    canvas.height = 600;
+    const context = canvas.getContext("2d");
+    this.setState({ canvas, context });
```

Comment:
don't use `setState` in `componentDidMount` because it forces component to double render on mounting. You can set initial state in constructor.